syntax = "proto3";

option csharp_namespace = "ProfTestium_TestService";


import "google/protobuf/duration.proto"; 
import "google/protobuf/timestamp.proto";


package greet;

service SessionSender{
	rpc GetSessionList(SessionSenderRequest) returns(SessionSenderReply);
}

message SessionSenderRequest{
	int32 UserID = 1;
}
message TestsSenderReply{
	int32 TestID = 1;
	string TestName = 2;
	}
message SessionSenderReply{
	repeated int32 SessionID = 1;
	repeated TestsSenderReply testssenderReply = 2;
	repeated google.protobuf.Timestamp SessionDate = 3;
	repeated google.protobuf.Duration DurationSession = 4;
	repeated bool IsSuccessful = 5;
	repeated string FailureReason = 6;
	repeated string SessionUrl = 7;
	repeated int32 Score = 8;
	repeated int32 MaxScore = 9;
}
